client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py

```diff
@@ -43,38 +43,58 @@ def process(self, instance):
         # find thumbnail path
         thumbnail_path = instance.data.get("thumbnailPath")
 
+        # set pathtoframes exclusion filters
+        frames_path_filters = ["thumb", "workfile"]
+
         found_reviewable = False
+        found_frames = False
+
         for representation in instance.data.get("representations", []):
             self.log.debug(pformat(representation))
 
-            if "shotgridreview" not in representation.get("tags", []):
+            # Skip unwanted representations
+            if any(filter_name for filter_name in frames_path_filters if filter_name in f".{representation['name']}"):
                 continue
 
             local_path = get_publish_repre_path(
                 instance, representation, False
             )
 
-            if f".{representation['ext']}" in VIDEO_EXTENSIONS:
-                found_reviewable = True
-                data_to_update["sg_path_to_movie"] = local_path
-                if (
-                    "slate" in instance.data["families"]
-                    and "slate-frame" in representation["tags"]
-                ):
-                    data_to_update["sg_movie_has_slate"] = True
-
-            elif f".{representation['ext']}" in IMAGE_EXTENSIONS:
-                found_reviewable = True
-                # Replace the frame number with '%04d'
-                path_to_frame = re.sub(r"\.\d+\.", ".%04d.", local_path)
-
-                data_to_update.update({
-                    "sg_path_to_movie": path_to_frame,
-                    "sg_path_to_frames": path_to_frame,
-                })
+            # Skip temp locations
+            if local_path.startswith("/tmp"):
```

Comment:
I'm concerned this is too platform-specific, maybe we should check against anatomy root path instead ?